assets/javascript/script.js

```diff
@@ -222,4 +223,49 @@ async function hightlightMarker(event) {
 }
```

Comment:
No need to fix spacing and indentation, I believe my code does follow all the rules so I don't think we need to waste time for these. I will take responsibility for the quality of my code!